packages/core/content-releases/server/src/migrations/database/5.0.0-document-id-in-actions.ts

```diff
@@ -13,6 +13,22 @@ type Knex = Parameters<Migration['up']>[0];
 export const addEntryDocumentToReleaseActions: Migration = {
   name: 'content-releases::5.0.0-add-entry-document-id-to-release-actions',
   async up(trx: Knex, db: Database) {
+    // Check if the table exists
+    const tableExists = await trx.schema.hasTable('strapi_release_actions');
+
+    if (!tableExists) {
+      // Create the table if it doesn't exist
+      await trx.schema.createTable('strapi_release_actions', (table) => {
+        table.increments('id').primary();
+        table.string('type');
+        table.string('content_type');
+        table.string('entry_document_id');
+        table.string('locale');
+        table.boolean('is_entry_valid');
+        table.integer('release_id').unsigned();
+      });
+    }
+
```

Comment:
I think you can actually just have it return rather than creating the table. If the table doesn't exist, there is nothing to migrate. And then Strapi will create the correct table structure on startup.

Reply:
oh I see! will do! :D